.gitignore

```diff
@@ -4,4 +4,8 @@ ffi/java-idsdk/.idea
 ffi/java-idsdk/target
 ffi/java-idsdk/**/*.class
 dist/
-.idea/
\ No newline at end of file
+.idea/
+ffi/dotnet-idsdk/.vscode
+ffi/dotnet-idsdk/**/.vs
+ffi/dotnet-idsdk/**/obj
+ffi/dotnet-idsdk/**/bin
```

Comment:
Add exclude for possibley object types in `lib` dir

Reply:
done